### PR TITLE
Fixes #14020: Update rudder server root to match modern packaging recommendation

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/14020

### CODE REVIEW

File: rudder-reports/SOURCES/Makefile [hunk 1]
```diff
@@ -16,23 +16,60 @@
 #
 #####################################################################################
 
-.DEFAULT_GOAL := localdepends
+.DEFAULT_GOAL := build
 
-RUDDER_VERSION_TO_PACKAGE = <put Rudder version or version-snapshot here>
+#RUDDER_VERSION_TO_PACKAGE = <put Rudder version or version-snapshot here>
+RUDDER_VERSION_TO_PACKAGE = 5.0.1
```
Comment (amousset) [inline rudder-reports/SOURCES/Makefile]: Same

File: rudder-reports/SOURCES/Makefile [hunk 2]
```diff
@@ -16,23 +16,60 @@
...
 
-./rudder-sources.tar.bz2:
-	$(WGET) -O rudder-sources.tar.bz2 https://www.rudder-project.org/archives/rudder-sources-${RUDDER_VERSION_TO_PACKAGE}.tar.bz2
+DESTDIR = $(CURDIR)/target
 
-./rudder-sources: ./rudder-sources.tar.bz2
+localdepends:
+
+rudder-sources.tar.bz2:
+	$(GET) rudder-sources.tar.bz2 http://repository.rudder.io/sources/$(RUDDER_VERSION_TO_PACKAGE)/rudder-sources-$(RUDDER_VERSION_TO_PACKAGE).tar.bz2 ""
```
Comment (amousset) [inline rudder-reports/SOURCES/Makefile]: https

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**